stockexchange/bot/management/commands/bot.py

```diff
@@ -0,0 +1,129 @@
+from django.core.management.base import BaseCommand
+from django.conf import settings
+import telebot
+from polygon import RESTClient
+from ...models import User, Portfolio
+import datetime
+from datetime import date
+from telebot import types
+import locale
+
+bot = telebot.TeleBot(settings.TOKEN)
+
+
+class Command(BaseCommand):
+    help = 'Stock exchange bot'
+
+    def handle(self, *args, **options):
+        bot.infinity_polling()
+
+
+@bot.message_handler(commands=["start"])
+def start(m):
+    u = User.objects.filter(id=m.chat.id)
+    if not u:
+        u = User(id=m.chat.id)
+        u.firstname = m.chat.first_name
+        u.secondname = m.chat.last_name
+        u.username = m.chat.username
+        try:
+            u.save()
+        except:
+            print(f'An error occurred while saving the user to the database: id {u.id}, username {u.username}')
```

Comment:
usually no one catch that exception. if DB is down, probably bot should be unavailable. DB falling is an exceptional situation.

but if you catching exception, it should be specific, not too broad.

also use logging instead of prints. prints are good for debbuging, not for production